packages/kbn-esql-validation-autocomplete/src/autocomplete/factories.ts

```diff
@@ -30,7 +30,10 @@ const allFunctions = statsAggregationFunctionDefinitions
   .concat(groupingFunctionDefinitions);
 
 export const TIME_SYSTEM_PARAMS = ['?t_start', '?t_end'];
-export const ADD_DATE_HISTOGRAM_SNIPPET = 'BUCKET($0, 10, ?t_start, ?t_end)';
+
+export const getAddDateHistogramSnippet = (histogramBarTarget = 50) => {
+  return `BUCKET($0, ${histogramBarTarget}, ?t_start, ?t_end)`;
```

Comment:
super nit: in the same file we have the `TIME_SYSTEM_PARAMS` array which define the t_start and t_end. We could reuse it here to avoid writing these variables here.

Reply:
Updated here  [`3f6f1b9` (#192038)](https://github.com/elastic/kibana/pull/192038/commits/3f6f1b97ce7b415ed2682ed3883ecc09c44eee11)